packages/compiler/src/server/tmlanguage.ts

```diff
@@ -160,12 +168,62 @@ const blockComment: BeginEndRule = {
   end: "\\*/",
 };
 
+const docCommentParam: MatchRule = {
```

Comment:
Seems like we should have at least one test for an unknown tag mapping to the new `entity.name.function.macro.tsp`

Reply:
it does have it with `@custom`